00-about-thursdays-at-ada/norms.md

```diff
@@ -1,19 +1,15 @@
 # Thursdays at Ada Norms
 
-* **Attendance at Thursdays at Ada is required for graduation** from the program. 
-  * We expect interns to be present for the entirety of the four hour Thursdays at Ada sessions. 
-  * We understand that sometimes things come up and external circumstances may cause you to miss part of or the entirety of a session. If you need to miss a CS Fundamentals or Career Services session, please send an email to [absent_internship@adadevelopersacademy.org
-  ](mailto:absent_internship@adadevelopersacademy.org
-  ) in advance when possible.
-  * Up to two excused absences are permitted throughout the course of Thursdays at Ada. This is across both Career Development and CS Fundamentals sessions.
+* **Attendance at Thursdays at Ada is optional for graduation** from the program. 
+  * We expect interns to be choose to engage with live Thursdays at Ada content as is most helpful to their learning needs.
 * CS Fundamentals assignments are a required part of your continuing education and graduation requirements. 
-  * **You need to complete 80% of all assigned work for CS Fundamentals**. 
-  * New topics and problem sets are released two weeks before they are due
+  * **You need to complete 80% of problem sets for CS Fundamentals**. 
+  * **You need to complete 100% of Interview Practice Questions for CS Fundamentals**.
+  * **You need to complete 5 peer-to-peer mock interviews for CS Fundamentals**.
+  * New topics and problem sets are released at least two weeks before they are due
   * New projects are released four weeks before they are due
```

Comment:
We might want to update this or delete as we don't have any standardized "due dates" for this cohort.